manifests/plugin.pp

```diff
@@ -129,8 +133,8 @@
     }
     default: {
       exec {"remove_plugin_${name}":
-        command => "${elasticsearch::plugintool} --remove ${module_dir}",
-        onlyif  => "test -d ${elasticsearch::plugindir}/${module_dir}",
+        command => "${elasticsearch::plugintool} --remove ${plugin_dir}",
+        onlyif  => "test -d ${elasticsearch::plugindir}/${plugin_dir}",
         notify  => $notify_service,
       }
     }
```

Comment:
This could be a bit dangerous having this as the default. I realise it's not changed in this commit but thought I'd mention it.